at_commons/lib/src/verb/notify_fetch_verb_builder.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:at_commons/src/verb/verb_builder.dart';
+
+/// NotifyFetchVerbBuilder generated the command to fetches the notification
+/// with the given notification-id
+///
+/// ```
+/// e.g. To fetch the notification with the notification-id - '21d617cd-fb8a-43e4-98df-7e1536818d59'
+///
+/// var notifyFetchVerbBuilder = NotifyFetchVerbBuilder()
+///                                       ..notificationId = '21d617cd-fb8a-43e4-98df-7e1536818d59';
+/// ```
+class NotifyFetchVerbBuilder implements VerbBuilder {
+  /// Notification Id to fetch the notification
+  String? notificationId;
+
+  @override
+  String buildCommand() {
+    var command = 'notify:fetch:';
+    if (notificationId != null) {
+      command += '$notificationId\n';
+    }
+    return command;
+  }
+
+  @override
+  bool checkParams() {
+    var isValid = true;
+    if (notificationId == null) {
```

Comment:
Check for empty string or invalid (eg contains spaces)

Reply:
Replaced the null check with isEmpty